packages/headless/src/controllers/core/facets/facet-search/facet-search.ts

```diff
@@ -72,11 +71,11 @@ export function buildGenericFacetSearch<T extends FacetSearchState>(
     },
 
     /** Selects a search result.*/
-    select(value: GenericFacetSearchResult) {
-      const facetValue = value.rawValue;
+    select(_value: GenericFacetSearchResult) {
+      // const facetValue = value.rawValue;
 
       dispatch(updateFacetOptions({freezeFacetOrder: true}));
-      dispatch(executeSearch(logFacetSelect({facetId, facetValue})));
+      // dispatch(executeSearch(logFacetSelect({facetId, facetValue})));
```

Comment:
Are those meant to be commented out?

Reply:
Yes! Sorry, those are changes that I processed in the second PR, I didn't want to forget them :)